src/utils/http/errors/APIError.js

```diff
@@ -0,0 +1,16 @@
+const http = require('http');
```

Comment:
Why are you using require instead of import?

Reply:
just copied from somewhere, changed it